core/assembly/MetadataLoadContext/Program.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace MetadataLoadContextSample
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {            
+            if (args.Length < 1)
+            {
+                Console.WriteLine("Usage: dotnet MetadataLoadContextSample.dll <assembly path>");
+                return 0;
+            }
+
+            string inputFile = args[0];
+
+            try
+            {
+                //get the array of runtime assemblies
```

Comment:
This is my first time reviewing a sample so perhaps @BillWagner or @Thraka can correct me here, but I think we need to capitalize comments and put a period on the end.

```suggestion
                // Get the array of runtime assemblies.
```

Reply:
General [comment conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#commenting-conventions) recommend the same, so i'll apply this.